packages/desktop/src/services/FileDownloadServiceElectron.ts

```diff
@@ -279,10 +381,10 @@ export default class FileDownloadServiceElectron
                 });
             });
 
-            req.on("error", async (err) => {
+            req.on("error", async (err: unknown) => {
```

Comment:
implicit or `Error`?
```suggestion
            req.on("error", async (err) => {
```

Reply:
resolved in [cfdf9b4](https://github.com/AllenInstitute/biofile-finder/pull/204/commits/cfdf9b40628c86077ac4ad73f158267bde3b5b49)